src/osd/OSD.h

```diff
@@ -1596,7 +1596,7 @@ class OSD : public Dispatcher,
    * sufficient.
    */
   void osdmap_subscribe(version_t epoch, bool force_request);
-  /** @} monc helpers */
+  /** @ monc helpers */
```

Comment:
This ends a Doxygen grouping.  See defgroup in https://www.stack.nl/~dimitri/doxygen/manual/commands.html   I think  the problem here is a missing **@{**.